resources/function_help/json/create_ramp

```diff
@@ -5,5 +5,5 @@
   "description": "Returns a gradient ramp from a map of color strings and steps.",
   "arguments": [ {"arg":"map","description":"a map of color strings and steps"},
                  {"arg":"discrete","optional":true,"description":"declare whether the color ramp is discrete"}],
```

Comment:
I think it might be good to know the default value/behavior, here

Reply:
It's "false"

Reply:
```suggestion
                 {"arg":"discrete","optional":true,"default":"false","description":"set this parameter to true to create a discrete color ramp"}],
```